client/stylesheets/_modal.scss

```diff
@@ -843,9 +843,68 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 	margin-top: 75px !important;
 }
 
+.modFeedbackPageTwo {
+	margin-top: 50px !important;
+	padding: 10px 40px !important;
+}
+
+.modFeedbackPageTwo h2 {
+	color: $color-spaceblue !important;
+	font-size: 2rem !important;
+	font-weight: 400 !important;
+	margin: 20px 0 !important;
+}
+
+.modFeedbackPageTwo1 {
+	margin-top: 40px;
+}
+
+.modFeedbackPageTwo1 ul {
```

Comment:
Merge rule `.modFeedbackPageTwo1 ul` with rule on line 858<br>Selector `modFeedbackPageTwo1` should be written in lowercase with hyphens